crates/rooch/src/commands/account/commands/mod.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 pub mod balance;
```

Comment:
I suggest this mod to be renamed to `inquire`? Never mind if it's only for inquire about account balance.